.pipelines/templates/e2e-test-kind.yaml

```diff
@@ -1,37 +1,33 @@
 parameters:
   - name: driverWriteSecrets
     type: object
+  - name: helmTests
+    type: object
 
 jobs:
   - ${{ each driverWriteSecret in parameters.driverWriteSecrets }}:
     - job:
       displayName: ${{ format('e2e_test_kind/driver-write-secrets={0}', driverWriteSecret) }}
-      timeoutInMinutes: 10
+      timeoutInMinutes: 15
```

Comment:
Do we need to increase the timeout? Are we seeing timeouts?

Reply:
It [failed](https://dev.azure.com/azure/secrets-store-csi-driver-provider-azure/_build/results?buildId=27078&view=logs&j=7f33e5bd-7764-5d8a-ba2e-506e078b9c3f) earlier. But let me revert it back and monitor since we created separate job for yaml testing.

Reply:
reverted back to 10m